desktop/src/renderer/components/app-document/app-document-editor/app-document-editor.tsx

```diff
@@ -14,35 +14,35 @@ export class AppDocumentEditor {
 
   private editorRef: HTMLStencilaEditorElement | null = null
 
-  @Prop() filePath: string
+  @Prop() documentId: string
 
   private file?: File
```

Comment:
I'm uneasy about the editor retaining a reference to a `File` (presumably the `File` that it was created for). For a new document, created in the app, there will be no `File` yet, if files are renamed then this could become stale. I see that `File` is used in `fileFormatToLanguage` but I suggest that it would be better to have a `private document: Document` instead since that has a `format` property (optionally for new documents as well).